nl_server/embeddings.py

```diff
@@ -242,6 +242,9 @@ def _detect_multiple_svs(self, query: str) -> vars.MultiVarCandidates:
           # so drop this candidate.
           continue
 
+        if not vars.deduplicate_svs(candidate):
```

Comment:
Why is this needed?

Reply:
To dedup SVs that show up in multiple parts of the candidate.  Updated test and added comment.